packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -203,11 +210,22 @@ class DateTimePicker extends React.Component {
     }
   }
 
+  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
Where is this method being used?
Since it is a static method can this be moved into the DatTimeUtils?

Reply:
It's a standard lifecycle method from [react](https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops)